packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHeadersHidden.4.doc.mdx

```diff
@@ -0,0 +1,6 @@
+import ColumnHeadersHidden from './ColumnHeadersHidden?dev-site-example';
+
```

Comment:
This should include a description of what this prop is used for and make sure consumers know they still have to provide column header labels, render their own column headers at the same widths as the columns, and ensure the two match.

The example itself should include that as well and there should be mention of it in the prop description.